bin/packages/get-babel-config.js

```diff
@@ -22,27 +22,33 @@ const overrideOptions = ( target, targetName, options ) => {
 const babelConfigs = {
 	main: Object.assign( {}, babelDefaultConfig, {
 		plugins,
-		presets: map( babelDefaultConfig.presets, ( preset ) =>
-			overrideOptions( preset, '@babel/preset-env', {
-				modules: 'commonjs',
-				corejs: '3',
-				useBuiltIns: 'usage',
-			} )
-		),
+		presets: [
+			[ '@babel/preset-typescript' ],
```

Comment:
add the TS preset to the package build babel config. It is my dream to remove all this seperate build stuff and let webpack compile it instead, I hope to get to this in the next few months but for now, this will have to do.